test/AutoRest.Shared.Tests/AutoRest.Shared.Tests.csproj

```diff
@@ -1,10 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```

Comment:
@ArthurMa1978 PR just removed this are we sure we want to add it back?

Reply:
I believe this change comes from merging. Let me revert it.